cmc-ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ContactDetailsMapper.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import uk.gov.hmcts.cmc.ccd.domain.ContactDetails;
+
+public class ContactDetailsMapper
+    implements Mapper<ContactDetails, uk.gov.hmcts.cmc.domain.models.legalrep.ContactDetails> {
+
+    @Override
+    public ContactDetails to(uk.gov.hmcts.cmc.domain.models.legalrep.ContactDetails contactDetails) {
+        if (contactDetails == null) {
```

Comment:
Please don't write methods that can return nulls this way. Either change the signature to return an `Optional` or do not call this method if the input is null.